library/item/item_group/src/main/java/org/quiltmc/qsl/item/group/mixin/client/CreativeInventoryScreenMixin.java

```diff
@@ -165,6 +173,18 @@ private void renderTabIcon(MatrixStack matrixStack, ItemGroup itemGroup, Callbac
 		if (this.quilt$isGroupNotVisible(itemGroup)) {
 			info.cancel();
 		}
+
+		this.quilt$renderTabIcon$matrices = matrixStack;
+		this.quilt$renderTabIcon$itemGroup = itemGroup;
+	}
+
+	@ModifyArgs(method = "renderTabIcon", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/item/ItemRenderer;renderInGuiWithOverrides(Lnet/minecraft/item/ItemStack;II)V"))
+	private void renderCustomTabIcon(Args args) {
```

Comment:
You can get the method args in a ModifyArgs; there's no reason for the fields you're using